### ISSUE TITLE
error handler no longer reports message/stack

### ISSUE THREAD

**Issue by BerkeleyTrue** [keymetrics/pmx#61]
Change 655af14 causes error messages reported to keymetrics to be without stack/message leaving the reported error almost useless (It at least still tells you that something happened). Unless the user is aware of what is going on they may even dismiss a potentially serious issue as a bug in keymetrics reporting, since the message is not obviously an error or fault message, but looks like a log message.

The culprit lies here: https://github.com/keymetrics/pmx/blob/master/lib/notify.js#L111 added back in June.

The new object created, while still having the stack/message props, does not have them on instance itself but on it's prototype. 

Later on, getOwnPropertyNames is used [here](https://github.com/keymetrics/pmx/blob/master/lib/notify.js#L19) to create yet another new object and the stack/message is lost.

There are several possible fixes to this. The easiest is to get rid of the line added in 655af14, and instead call jsonize first, than add the addition properties (url, component, etc...).

```js
Notify.expressErrorHandler = function() {
  var self = this;

  Options.configureModule({
    error : true
  });

  return function errorHandler(err, req, res, next) {
    if (res.statusCode < 400) res.statusCode = 500;
    
    err =  jsonize(err);

    //debug(err.stack || err);

    err.url = req.url;
    err.component = req.url;
    err.action = req.method;
    err.params = req.body;
    err.session = req.session;

    Transport.send({
      type  : 'process:exception',
      data  : err
    }, true);
    return next(err);
  };
};
```

**Comment by Unitech** [comment]
Thanks for reporting, I just removed the line and re published PMX (0.5.9)

**Comment by BerkeleyTrue** [comment]
@Unitech Thanks for fixing that. Although now that the error is mutated again, a minor version bump should have been made to keep with Semvar.
